package.json

```
@@ -3,13 +3,15 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "ts-node src/server.ts"
   },
   "author": "",
   "license": "ISC",
   "description": "",
   "dependencies": {
     "@types/node": "^22.7.4",
+    "graphql": "^16.9.0",
```

geralmente fixamos as versões de dependências para não termos problemas com atualizações
```suggestion
    "graphql": "16.9.0",
```

---

Show, vou fazer a alteração.

---

faltou fixar as versões

---

Fixadas